apps/package.json

```diff
@@ -297,6 +301,7 @@
     "rehype-raw": "^4.0.0",
     "rehype-react": "^3.1.0",
     "rehype-sanitize": "^2.0.2",
+    "remark-parse": "~8.0.3",
```

Comment:
For my understanding, can you help me understand how you picked out these "transient dependency" versions (is that the right word)?

Reply:
Cool, TIL about `yarn why`! Two more questions for you for my learning:
1. How did you identify we use remark-parse 8.0 features? Test failures?
2. How did you identify which packages we might need to add direct dependencies for? Just anything where a major version dependency changed in `yarn.lock` after doing this?